src/calculateRentalCost.js

```diff
@@ -31,17 +31,16 @@ function calculateRentalCost(days) {
   const dayRate = 40;
   const threeDayDiscount = 20;
   const sevenDayDiscount = 50;
+  const cost = (days * dayRate);
 
-  if (days >= 3 && days < 7) {
-    return (days * dayRate) - threeDayDiscount;
-  }
-
-  if (days >= 7) {
-    return (days * dayRate) - sevenDayDiscount;
+  if (days < 3) {
+    return cost;
   }
 
-  if (days < 3) {
-    return days * dayRate;
+  if (days >= 3 && days < 7) {
```

Comment:
`days >= 3` this condition is always true here
because you would have already returned a value in the prev `if`